geometry_msgs/Point[] points
---
bool timed_out
bool success
---
